foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -38,70 +37,68 @@ Follow these steps to add the install the extension in Experience Platform Launc
    implementation 'com.adobe.marketing.mobile:assurance:1+'
    ```
 
-2. Import the Project Griffon libraries with the other SDK libraries:
+2. Import the Assurance library with the other Mobile SDK libraries:
```

Comment:
nit: Import the Assurance library along with the other Adobe Mobile SDK libraries: